uportal-war/src/main/webapp/WEB-INF/jsp/Marketplace/portlet/entry.jsp

```diff
@@ -387,19 +129,78 @@
             </div>
             <div class="col-sm-11">
                 <div class="marketplace_description_title">
-                    <h1>${marketplaceEntry.title}</h1>
+                    <h1 class = "fivepx-margin-top">${marketplaceEntry.title}</h1>
                 </div>
                 <div class="marketplace_description_body">
                     <p>${marketplaceEntry.description}</p>
                 </div>
             </div>
-        </div>
+			<div class="${n}go_button margin-top col-xs-12">
+				<div class="btn-group marketplace_button_group">
+					<a href="${marketplaceEntry.renderUrl}" id="marketplace_go_button" class="btn btn-default btn-sm marketplace_dropdown_button" role="button">
+						<spring:message code="go" text="Go"/>
+					</a>
+					<button type="button" class="btn btn-default btn-sm dropdown-toggle marketplace_dropdown_button" data-toggle="dropdown">
+						<spring:message code="options" text="Options"/>
+						<span class="caret"></span>
+					</button>
+					<ul class="dropdown-menu marketplace_dropdown_menu" role="menu">
+					</ul>
+					<button type="button" class="btn btn-default btn-sm" data-toggle="modal" data-target="#${n}-feedback-modal"><span class = "fa fa-pencil"></span><span class = "padding-left">Rate this application</span></button>
+				</div>
+  				<!-- Modal -->
+				<div class="modal fade" id="${n}-feedback-modal" role="dialog">
```

Comment:
Review this portlet form now appears in a dialog, as opposed to displaying in the page on load.